### PR TITLE
[dependencies] Add support for django 3.0 #47

### LINKED ISSUE #47: Add support for django 3.0
Add support for django 3.0, drop django version older than 2.2.

### PR DESCRIPTION
**Pull request by Weirdo914**
Added support for django 3.0 and updated dependencies
Closes #47

### CODE REVIEW

**Comment by atb00ker** [comment]
Please read: https://github.com/openwisp/django-loci/pull/50#issuecomment-576162743

**Comment by Weirdo914** [comment]
I was going to pick this task up after my current task so I was doing some work in advance. I will take care next time.

File: tests/urls.py [hunk 1]
```diff
@@ -3,9 +3,7 @@
 from django.conf.urls.static import static
 from django.contrib.staticfiles.urls import staticfiles_urlpatterns
 
-from openwisp_utils.admin_theme.admin import admin, openwisp_admin
-
-openwisp_admin()
+from openwisp_utils.admin_theme.admin import admin
```
Comment (nemesisdesign) [inline tests/urls.py]: change this to `from django.contrib import admin` and use `isort -y` to reorder the imports

**Comment by nemesisdesign** [comment]
Thanks!

**Pull request merged**